### PR TITLE
docs - document unified2 removal - v1 

### PR DESCRIPTION
**Pull request by jasonish**
First attempt at documenting the removal of unified2 and how to move forward.

### CODE REVIEW

**Comment by jasonish** [comment]
Sorry about the CI, its not something I can recreate locally in the same distributions. Still looking into it.

**Comment by jasonish** [comment]
> Sorry about the CI, its not something I can recreate locally in the same distributions. Still looking into it.

Resolved.

File: doc/userguide/upgrade/unified2.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+:orphan: Document not referenced in a toctree, so add this.
```
Comment (victorjulien) [inline doc/userguide/upgrade/unified2.rst:1]: Should we just add it to a toc somewhere?

Comment (jasonish) [reply doc/userguide/upgrade/unified2.rst:1]: I tried but it never looked right..  I have another idea to try.

**Comment by jasonish** [comment]
Without re-working the whole upgrade page, this is the best I can come up with...
Screenshot from 2020-09-09 09-14-37

**Review by victorjulien — APPROVED** [review]

**Comment by victorjulien** [comment]
Merged in #5405, thanks!

**Pull request closed without merging**